lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -606,6 +607,7 @@ SILPassPipelinePlan::getSILOptPreparePassPipeline(const SILOptions &Options) {
   }
 
   P.startPipeline("SILOpt Prepare Passes");
+  P.addMandatoryInlining();
```

Comment:
You shouldn't need another run here.

Reply:
I thought this was needed but you're right (it isn't).